services/merak-network/utils/constant.go

```diff
@@ -24,4 +24,4 @@ const SECURITYGROUP string = "securityGroup:"
 const NETCONFIG string = "netconfig:"
 const NODEGROUP string = "nodeGroup:"
 
-const ALCORURL string = "10.213.43.77"
+var ALCORURL = "10.213.43.77"
```

Comment:
@yanmo96 why you still need assign '10.213.43.77' for `ALCORURL`? Please remove it or assign an empty string to it if it necessary.

Reply:
ok